Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp

```diff
@@ -361,7 +361,7 @@ namespace ProcessContainers {
         }
 
         if (_useSpecFile) {
-            TRACE(ProcessContainers::ProcessContainerization, (_T("path set to %s for name %s"), _path.c_str(), _name.c_str()));
+            TRACE(Trace::Information, (_T("path set to %s for name %s"), _path.c_str(), _name.c_str()));
```

Comment:
changing everything from `ProcessContainers::ProcessContainerization` to `Trace::Information` is okey for now.